guides/common/modules/snip_host-registration-steps.adoc

```diff
@@ -94,7 +131,7 @@ You can use the ID of any {SmartProxyServer} that you configured for host regist
 +
 Include `{ "force": true }` to register a host that has been previously registered to a {SmartProxyServer}.
 endif::[]
-+
+
```

Comment:
I don't think this is correct since the diff of the rendered result now shows `+` before `Include` is rendered literally.

Reply:
Fixed. Aligned those paras to 1st point.